MediaRecorder.bs

```diff
@@ -286,11 +295,30 @@ interface MediaRecorder : EventTarget {
     <li>Set |recorder|'s {{state}} to {{recording}}, and run the following steps
     in parallel:
     <ol>
-      <li>Start recording all tracks in |tracks| using the {{MediaRecorder}}'s
-      current configuration, and gather the data into a {{Blob}} <var>blob</var>
-      and queue a task, using the DOM manipulation task source, to
-      <a>fire an event</a> named <a>start</a> at <var>recorder</var>, then return
-      <code>undefined</code>.</li>
+      <li>As soon as data is available on all tracks in |tracks|, if the
+      |recorder|'s current configuration does not already specify the codec and
+      parameters to use, the User Agent MUST specify this. The User Agent MAY
+      take the content of the first frame into account when making this
+      decision. The User Agent MUST then start recording with the configuration
+      and gather the data into a {{Blob}} <var>blob</var>. As soon as recording
+      starts, queue a task, using the DOM manipulation task source, to run the
+      following steps:
+      <ol>
+        <li>Set |recorder|'s {{mimeType}} to reflect the current
+        configuration, including codec and parameter information.</li>
+        <li><a>Fire an event</a> named <a>start</a> at <var>recorder</var>, then
+        return <code>undefined</code>.</li>
+      </ol>
+      <div class="note">
+        By looking at the content of the tracks when recording start, the User
+        Agent could take frame information into account such as if the video is
+        high or low resolution. If the MediaStreamTrack is a remote track
```

Comment:
Resolution is a bad example, because it can change on the fly at any time. You also don't mention what it could be used for, so I'd rather just leave it out.